dev.rollup.config.js

```diff
@@ -1,8 +1,6 @@
-/* eslint-env node */
```

Comment:
Does the node environment come in from one of the recommended presets by default?  It must as eslint isn't complaining if I reference a node global such as process.  Was just wondering where it comes from.

Reply:
So the node env is coming from typescript here. In our `tsconfig.json` I explicitly list the included types as `['jasmine']` which excludes node types and that's how I'm able to exclude node globals from our source code. This config doesn't apply to files outside the `src` dir and therefore TS will pull in `@types/node` from node_modules and so eslint will recognize the node globals in the rollup config.